tensorboard/plugins/custom_scalar/README.md

```diff
@@ -0,0 +1,175 @@
+# The Custom Scalar Dashboard
+
+## The Problem this Dashboard Solves
+
+The (other) scalar dashboard shows line charts for data collected from scalar
```

Comment:
maybe s/other/original/ here to better capture that the other scalar dashboard is just "the" scalar dashboard to most users?

Reply:
original SG. Done.